web/src/resources/TagResource.ts

```diff
@@ -264,3 +264,10 @@ export async function getSecurityDetails(
   assertHttpCode(response.status, 200);
   return response.data;
 }
+
+export async function createTag(org: string, repo: string, tag: string, manifest: string) {
+    await axios.put(
+      `/api/v1/repository/${org}/${repo}/tag/${tag}`,
+      {manifest_digest: manifest},
+    );
```

Comment:
could we also add type checking, assert on status code and try catch in case of error - to keep it consistent with  existing axios requests?

Reply:
This is now being handled by react query, we should consider removing that code from the other axios requests.